### PR TITLE
Checkstyle fix

### PR DESCRIPTION
**Pull request by jGleitz**

### CODE REVIEW

**Comment by acBerger** [comment]
lgtm

**Comment by acBerger** [comment]
lgtm

File: test/Checkstyle.xml [hunk 1]
```diff
@@ -64,7 +64,9 @@
     </module>
     <module name="SummaryJavadoc">
       <property name="severity" value="warning"/>
-      <property name="forbiddenSummaryFragments" value="^.{20,100}$"/>
+      <property name="forbiddenSummaryFragments" value="^[^\.]{0,20}\..*$"/>
+      <message key="summary.javaDoc" value="Zusammenfassung zu kurz (mindestens 20 Zeichen!)."/>
```
Comment (RomanLangrehr) [inline test/Checkstyle.xml]: Why is this in german?

**Comment by jGleitz** [comment]
Because the messages that were in there already were in German. So I stuck to that, without thinking about it. But you're right, they should be in English

**Comment by jGleitz** [comment]
Translated the messages.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**